roles/openshift_metrics/tasks/install_heapster.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- shell: >
+    {{ openshift.common.client_binary }} -n {{ openshift_metrics_project | quote }}
+    --config={{ mktemp.stdout }}/admin.kubeconfig
+    get rc heapster -o jsonpath='{.spec.replicas}' || echo 0
```

Comment:
Updated where I could.  I have one case where register produces a list.  A following task pulls values by index from the list so this method doesnt quite work so I had to leave the echo